### DISCUSSION TITLE
Windows Binary

### DISCUSSION THREAD

**Discussion by JayTea144** [mxpv/podsync#481] [Q&A] [answered]
For the fork of this "fqx/podsync-with-yt-dlp" - the README.md file says:
"Runs on Windows, Mac OS, Linux, and Docker."
[https://github.com/fqx/podsync-with-yt-dlp#readme](url)

How do I get/make a windows file for the fork above?

Thanks.

**Comment by JayTea144** [comment, marked as answer]
I was able to implement a work around.

I simply renamed "yt-dlp.exe" to "youtube-dl.exe".  Podsync was happy and didn't know any difference.
Now my download speeds are way faster.

Thanks.

**Comment by Elycoo** [comment]
For Linux I cheated it using those lines taken from the docker file:
```
sudo wget -O /usr/bin/yt-dlp https://github.com/yt-dlp/yt-dlp/releases/latest/download/yt-dlp && \
sudo chmod +x /usr/bin/yt-dlp && \
sudo ln -s /usr/bin/yt-dlp /usr/bin/youtube-dl
```
You may need to change the `/usr/bin/youtube-dl` to `/usr/local/bin/youtube-dl` in the last command and remove the file that has been there. Use `which youtube-dl` to find out.
